pkgs/development/python-modules/bsddb3/default.nix

```diff
@@ -15,8 +15,10 @@ buildPythonPackage rec {
     sha256 = "70d05ec8dc568f42e70fc919a442e0daadc2a905a1cfb7ca77f549d49d6e7801";
   };
 
+
```

Comment:
Keep formatting change in another commit ideally.

Reply:
Will keep this in mind for further PRs